cage.c

```diff
@@ -630,5 +630,8 @@ main(int argc, char *argv[])
 	/* This function is not null-safe, but we only ever get here
 	   with a proper wl_display. */
 	wl_display_destroy(server.wl_display);
+	wlr_scene_node_destroy(&server.scene->tree.node);
+	wlr_renderer_destroy(server.renderer);
+	wlr_allocator_destroy(server.allocator);
```

Comment:
At initialization, allocation is created after renderer. Should it be destroy before renderer?